internal/olm/client.go

```diff
@@ -39,16 +39,12 @@ import (
 	"k8s.io/client-go/rest"
 )
 
-var (
-	olmOperatorKey     = types.NamespacedName{Namespace: olmresourceclient.OLMNamespace, Name: "olm-operator"}
-	catalogOperatorKey = types.NamespacedName{Namespace: olmresourceclient.OLMNamespace, Name: "catalog-operator"}
-	packageServerKey   = types.NamespacedName{Namespace: olmresourceclient.OLMNamespace, Name: "packageserver"}
-)
-
 type Client struct {
 	*olmresourceclient.Client
 	HTTPClient      http.Client
 	BaseDownloadURL string
+	// OLM Namespace
+	Namespace string
```

Comment:
This namespace should only be known by the manager and passed into methods on `Client`.

Reply:
removed this

Reply:
Can you re-order these arguments so they read `namespace, version string`? `namespace` is more "contextual" than `version` so should be closer to `ctx` IMO.

Reply:
fixed